doc/bql.rst

```diff
@@ -130,17 +130,16 @@ Populations
 +++++++++++
 
 A population specifies which columns in a table should be modeled or ignored.
-For those that are modeled, it specifies whether they should be modeled as
-numerical or nominal.
+For those that are modeled, it specifies their statistical type.
 
 .. index:: ``GUESS SCHEMA``
 
 ``GUESS SCHEMA FOR <name>``
 
    Guess a population schema for the table *name*. A schema maps the columns in
    *name* to heuristically guessed statistical types. Statistical types can be
-   guessed to be NOMINAL or NUMERICAL. Columns can also be guessed to be ignored
-   (IGNORE).
+   guessed to be nominal or numerical. Columns can also be guessed to be
+   ignored.
```

Comment:
No particular preference here, but I feel like it may be appropriate to formally specify what statistical types `GUESS` might use.  (Not the heuristics it uses, just the statistical types.)